fed/_private/compatible_utils.py

```diff
@@ -66,6 +66,15 @@ def _get_gcs_address_from_ray_worker():
         return ray.worker._global_node.gcs_address
 
 
+def wrap_kv_key(job_name, key):
+    """Add an prefix to the key to avoid conflict with other jobs.
+    """
+    if (isinstance(key, bytes)):
```

Comment:
But we can't restrict the KV `get` and `put` to only use the `str` type key.

Do you mean changing current KV accessing to use `str` type only ?

Reply:
I meant:
```python
def wrap_kv_key(job_name, key):
    """Add an prefix to the key to avoid conflict with other jobs.
    """
    assert isinstance(key, str)
```